src/Cfg.h

```diff
@@ -40,11 +40,6 @@
 #define BENCHMARK_RESULTS()
 #endif
 
-#define LOG_LEVEL_NONE  0
-#define LOG_LEVEL_INFO  1
-#define LOG_LEVEL_WARN  2
-#define LOG_LEVEL_DEBUG 3
```

Comment:
These four lines where added when I merged the develop branch into the `wip/metronome` branch and so these four lines should not be deleted. Also all the code changes associated with this merge should not be deleted. I have posted discussions about a fix over on this thread http://piano-booster.2625608.n2.nabble.com/Add-code-for-the-PianoBooster-metronome-td7572964.html